tests/e2e/convolution/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# End-to-end convolution 2d tests.
+
+load("//build_tools/bazel:iree_e2e_generated_test.bzl", "iree_generated_e2e_test")
+
+package(
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+py_binary(
+    name = "generate_e2e_conv2d_tests",
+    srcs = ["generate_e2e_conv2d_tests.py"],
+)
+
+###########################################################################
+##
+## LLVMCPU backend
+##
+###########################################################################
+
+
+# LLVMCPU, non-data-tiling, no microkernels
```

Comment:
This comment will be stale once the defaults for data tiling and microkernels change (if they haven't already)

How about dropping that from the comment and just having this test "default flags"? Otherwise, you could also explicitly set the flags to false.